src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -163,14 +170,24 @@ public void updateFilteredPersonList(Predicate<Person> predicate) {
         requireNonNull(predicate);
         filteredPersons.setPredicate(predicate);
     }
-
+    /**
+     * Updates an unmodifiable view of the list of {@code Event} backed by the internal list of
+     * {@code versionedAddressBook}
+     */
     @Override
     public void updateFilteredEventList(Predicate<Event> predicate) {
         requireNonNull(predicate);
         filteredEvents.setPredicate(predicate);
     }
 
-
+    /**
+     * Updates the reference between Event and the persons it is tagged with.
```

Comment:
Small grammer change

```suggestion
     * Updates the reference between an {@code Event} and the persons it is tagged with.
```